circuits/cpp/src/aztec3/circuits/rollup/root/.test.cpp

```diff
@@ -16,6 +16,7 @@
 #include "index.hpp"
 #include "init.hpp"
 #include "c_bind.h"
+#include "aztec3/circuits/rollup/test_utils/utils.hpp"
```

Comment:
duplicated import

Reply:
Addressed in https://github.com/AztecProtocol/aztec3-packages/pull/342/commits/0091cd760999ae8ebc760ef640393658ccb36a51